### ISSUE TITLE
Extract web_connectivity data for Netalitica

### ISSUE THREAD

**Issue by hellais** [ooni/ooni.org#333]
This data analysis request comes from Igor from Netalitica.

Here it is:

Basically, I need data for 2 months (December 2019 -January 2020) from
results of net connectivity tests conducted in the countries which test
lists we have updated. I will then analyze this data to determined how
many new blocked websites were discovered. 

May I ask you to extract for me the results from the network connectivity tests
for Decemeber 2019 and January 2020 for the following countries: Egypt
Yemen, Belarus, Saudi Arabia, Syria, Pakistan, Tunisia, Sudan, UAE,
Bahrain, Morocco, Jordan, China, Uzbekistan, Turkmenistan, Russia,
Kazakhstan, Turkey and Iran.

**Comment by hellais** [comment]
I am working on this today.

**Comment by hellais** [comment]
I have dug into the measurements and came up with the following charts.

The target countries, with their relative country code are:
Egypt (EG)
Yemen (YE)
Belarus (BY)
Saudi Arabia (SA)
Syria (SY)
Pakistan (PK)
Tunisia (TN)
Sudan (SD)
United Arab Emirates (AE)
Bahrain (BH)
Morocco (MA)
Jordan (JO)
China (CN)
Uzbekistan (UZ)
Turkmenistan (TM)
Russia (RU)
Kazakhstan (KZ)
Turkey (TR)
Iran (IR)

Both charts are made aggregating the counts on a rolling week basis.

The first one looks at the number of unique websites which we have confirmed blocked in the target countries.

The caveaut with this is that we don't confirm blocking for all countries, either because we don't know how the blockpage looks like in that country or because that country does not rely on blockpage based blocking.

Below is the chart for that:
![Confirmed Count](https://user-images.githubusercontent.com/424620/74327871-38639800-4d8d-11ea-9550-2e0e02d5096c.png)

I also produced a chart which looks at anomalous measurements for all the target countries. The caveaut with this approach is that it's very sensitive to false positives as even a single false positive in the weekly window will result in a count in the window:

![Not False Count](https://user-images.githubusercontent.com/424620/74328029-7d87ca00-4d8d-11ea-8b45-94ee7d79d323.png)

For both of the charts another caveaut to keep in mind is that the number of sites which we see to be blocked are also impacted by the number of measurements we get from the region in that particular time window.
It may be a mistake to assume that seeing less "confirmed blocked" in a given window of time means that there is less blocking then or that our testing lists are less good, it may also simply be that we have less testing in the region in that particular time.

Here are the raw CSV files used to produce the charts above as well as the script used to generate the CSV files: https://gist.github.com/hellais/2f988a87d658373bed3a1a57793921e5.

The charts were made using tableau.
